README.md

```diff
@@ -20,10 +20,20 @@ Godel: Austrian logician and mathematician http://en.wikipedia.org/wiki/Kurt_G%C
   cd ..
   catkin_make
   ```
+  
+### Application can work at two modes:
+-  Local mode
+  -  All the ROS nodes run at the same machine.
+-  Network mode
+  -  ROS nodes run at two different machine: Server and Client.
+  -  Client runs ROS nodes "RVIZ" and "point_cloud_generator_node".
+  -  Server runs all other ROS nodes except two nodes running on client.
```

Comment:
Let me provide some clarification on the differences between simulated and real robot modes:
- In the context of this project, the **simulated robot** is just a ros node (not rviz) under the name of **industrial_robot_simulator**.  This node provides all the ros topics, services and actions that a **robot driver node** would provide if a physical robot was being used.
- When running in _real robot mode_, it means that there is a driver ros node(s) which is streaming robot command data to a physical robot controller through a socket connection.  This robot driver node(s) has the same ROS interface as the **simulated robot node** (same topics, services and actions) so ROS can't tell them apart from one another.  Furthermore, rviz is just visualization and User Front-end and plays no further role, in fact, you could run potentially run the entire blending software without rviz if you have a way to fake user interaction; as your node already does.
- In regards to RobotStudio, it is just software that runs outside the ROS ecosystem and emulates the robot controller.  Since the communication between the ros driver node and either the physical robot controller or RobotStudio emulator is done through a socket then both of these configurations are valid **real_robot** modes as to ros there isn't a difference.